python/tests/test_tree_stats.py

```diff
@@ -577,16 +564,19 @@ def test_single_tree(self):
         ts = msprime.simulate(6, random_seed=1)
         self.verify(ts)
 
+    @unittest.skip("inconsistent nan issues")
```

Comment:
Hmm, looks like this is skipping nearly all the tests for every stat now, not just node stats. I guess this is a sign we should resolve the inconsistency before merging?